transport/handler_server.go

```diff
@@ -332,7 +333,10 @@ func (ht *serverHandlerTransport) HandleStreams(startStream func(*Stream), trace
 	ctx = metadata.NewIncomingContext(ctx, ht.headerMD)
 	ctx = peer.NewContext(ctx, pr)
 	s.ctx = newContextWithStream(ctx, s)
-	s.trReader = &recvBufferReader{ctx: s.ctx, recv: s.buf}
+	s.trReader = &transportReader{
+		reader:        &recvBufferReader{ctx: s.ctx, recv: s.buf},
+		windowHandler: func(int) {},
```

Comment:
No. s.requestRead is set to an empty function above.
windowHandler closure is now part of transportReader and that should be an empty function too for handler_server. This is the reason I earlier set it directly to recvBufferReader (since we didn't need to do any windowUpdates), but forgot about the type assertion I added later for error checking.